### DISCUSSION TITLE
cloudflare next-on-pages edge runtime not working with prisma

### DISCUSSION THREAD

**Discussion by VanThanh12195** [prisma/prisma#23527] [Q&A]
### Bug description

![image](https://github.com/prisma/prisma/assets/14349308/c587ac61-a911-41d4-ab5b-35f25e76ffd8)
Hello, I try to follow this to set up prisma edge runtime. But can not get it to work.

I have another question: I have a postresql database on Azure, can I still use PlanetScale Serverless Driver for this database or I should use the PlanetScale databse for this driver to work?

Is there anyway to have prisma work on edge runtime with cloudflare next-on-pages without using Prisma Accelerate.

Thank you

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @VanThanh12195 👋 

Thank you for raising this question.

> Hello, I try to follow this to set up prisma edge runtime. But can not get it to work.

Can you please provide specific details about the error message and the steps you took to deploy to Cloudfare Pages?

> I have a PostgreSQL database on Azure, can I still use PlanetScale Serverless Driver for this database or I should use the PlanetScale database for this driver to work?

The PlanetScale Serverless Driver is designed to work with PlanetScale databases. It's not designed to work with a PostgreSQL database hosted on Azure. See a list of compatible database drivers [here](https://www.prisma.io/docs/orm/prisma-client/deployment/edge/overview#which-database-drivers-are-edge-compatible).

> Is there anyway to have prisma work on edge runtime with cloudflare next-on-pages without using Prisma Accelerate.

Yes there is. You can take a look at this [section](https://www.prisma.io/docs/orm/prisma-client/deployment/edge/deploy-to-cloudflare) of the documentation to learn how to deploy an app with Prisma ORM to Cloudfare pages or workers.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
